lib/magic-admin/resource/user.rb

```diff
@@ -34,16 +34,74 @@ def initialize(magic)
       # Arguments:
       #   issuer: Extracted iss component of DID Token generated by a Magic user
       #     on the client-side.
+      #   wallet_type: The type of wallet to retrieve.  To query specific wallet(s),
+      #     the value passed must be consistent with the enumerated values in
+      #     MagicAdmin::Resource::WalletType. ALL wallets will be returned if wallet_type=ANY
+      #     is passed. If the wallet_type is None or does not match any WalletType
+      #     enums, then no wallets are returned.
       #
       # Returns:
       #   Metadata information about the user
-      def get_metadata_by_issuer(issuer)
+      def get_metadata_by_issuer_and_wallet(issuer, wallet_type)
```

Comment:
FWIW, it might be useful to default `wallet_type` here to `NONE` or `ANY` if nothing is passed.

Reply:
Or it could also raise an error and show a message if nothing is passed as ```wallet_type```. But this is depending on what the desired behavior is.

Since I don't have context of this repo I don't know if the desired behavior is to default to something on behalf of the user, or make them send a ```wallet_type``` to the function 😅 . So just throwing ideas out there

Reply:
Thanks for the reviews both! Good callout on the default type - I created one and it helped consolidate logic 🙏🏼. Rather than creating three new methods, I updated the function to take in an optional wallet_type param.